.github/workflows/issue-to-project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add new issues to project
+
+on:
+  issues:
+    types:
+      - opened
+
+permissions:
+  issues: write
+  repository-projects: write
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.5.0
```

Comment:
https://github.com/actions/add-to-project/releases has `v1.0.2` already ... any reason why we want to use an older version?